docs/source/markdown/podman-network-create.1.md

```diff
@@ -28,6 +28,8 @@ resolution.
 Driver to manage the network. Currently `bridge`, `macvlan` and `ipvlan` are supported. Defaults to `bridge`.
 As rootless the `macvlan` and `ipvlan` driver have no access to the host network interfaces because rootless networking requires a separate network namespace.
 
+The `macvlan` and `ipvlan` drivers require the `--subnet` option, DHCP is currently not supported.
```

Comment:
This is not correct. It should say with netavark network backend. You can still use cni with dhcp just fine.
Also ipvlan is not supported at all with netavark:
```
$ bin/podman network create -d ipvlan 
Error: unsupported driver ipvlan: invalid argument
```